mopidy/internal/playlists.py

```diff
@@ -84,7 +84,7 @@ def parse_pls(data):
         if section.lower() != 'playlist':
             continue
         for i in range(cp.getint(section, 'numberofentries')):
-            yield cp.get(section, 'file%d' % (i + 1))
+            yield cp.get(section, 'file%d' % (i + 1)).strip('\"')
```

Comment:
We probably need to strip off single and double quotes.

Reply:
Maybe I'm overthinking this but: `re.sub(r'^([\'"])(.*)(\1)$', r'\2', '"test"')` - assuming we care about the quota types matching and being balanced. I.e. how forgiving do we want to be?

Reply:
@adamcik That would be the suspenders+belt+parachute solution I guess :)
I'm fine with whichever you decide is best. In the meantime I've amended and forced-pushed an updated PR with both ' and " stripped.

Reply:
Either is fine with me to be honest, given the weirdness that comes in some playlists I suspect more lenient might be better even if it never ends up triggering.